docs/development/plugins/data/server/kibana-plugin-plugins-data-server.iindexpattern.md

```diff
@@ -21,3 +21,9 @@ export interface IIndexPattern
 |  [title](./kibana-plugin-plugins-data-server.iindexpattern.title.md) | <code>string</code> |  |
 |  [type](./kibana-plugin-plugins-data-server.iindexpattern.type.md) | <code>string</code> |  |
 
+## Methods
+
+|  Method | Description |
+|  --- | --- |
+|  [getTimeField()](./kibana-plugin-plugins-data-server.iindexpattern.gettimefield.md) |  |
```

Comment:
This doesn't look related

Reply:
Yea it's not, it's a forgotten doc update from the data plugin, but I saw it too late and it will need to be pushed to master at some point anyway